apps/backend/src/mcp/tools/system_monitoring.py

```
@@ -4,64 +4,154 @@
 
-async def get_api_client() -> APIClient:
-    """Get or create global API client"""
-    global _api_client
-    if _api_client is None:
-        _api_client = APIClient()
-    return _api_client
 
-logger = logging.getLogger(__name__)
+class SystemMonitoringOperations:
+    """Utility class for system monitoring operations to eliminate duplicate collection methods."""
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)